ci/kind/test-metrics-doc-kind.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The script runs kind e2e tests with different traffic encapsulation modes.
+
+set -eo pipefail
+
+METRICS_TMP_DOC=$(mktemp /tmp/metricsdoc.XXXXXX.md)
+
+function exit_handler() {
+    echo "Cleaning up..."
+    if [ -f $METRICS_TMP_DOC ]; then
+        rm -rf $METRICS_TMP_DOC
+    fi
+}
+
+trap exit_handler INT EXIT
+
+MAKE_CMD=$(dirname $0)"/../../hack/make-metrics-doc.sh"
+METRICS_DOC=$(dirname $0)"/../../docs/prometheus-integration.md"
+
+cp -v $METRICS_DOC $METRICS_TMP_DOC
+$MAKE_CMD $METRICS_TMP_DOC
+cmp -s $METRICS_DOC $METRICS_TMP_DOC
+result=$?
+if [ $result -ne 0 ]; then
+    echo "Error: Prometheus metrics document should be updated"
```

Comment:
This message will be displayed by CI in case of failure and I think we should make it more useful for users. For example:
```bash
    echo "Error: Prometheus metrics document should be updated"
    echo "You can update it by building the Antrea Docker image locally (with `make`), running ./hack/make-metrics-doc.sh and committing the changes"
```